### PR TITLE
test delegating Appveyor to external worker (a.k.a ROBveyor)

### PR DESCRIPTION
**Pull request by ThomasLecocq**
small changes and test to add my own machine to build jobs https://github.com/obspy/obspy/pull/2567

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # AppVeyor.com is a Continuous Integration service to build and run tests under Windows
 # see https://github.com/ogrisel/python-appveyor-demo/blob/master/appveyor.yml
 
+build_cloud: PC1577-AS
+image: Windows
```
Comment (megies) [inline appveyor.yml]: looks like appveyor didnt even start a build with these changes? @ThomasLecocq 
Anyway the original PR looks good, no?

Comment (megies) [reply appveyor.yml]: Ah, maybe just glitched out, starting to build now after manually retriggering appveyor

**Comment by ThomasLecocq** [comment]
currently struggling starting the appveyor host service on my win machine, that's why... WIP

File: appveyor.yml [hunk 2]
```diff
@@ -95,6 +95,8 @@ on_success:
           python setup.py sdist bdist_wheel bdist_wininst bdist_egg
           Invoke-Expression "twine upload -u $env:PYPI_USERNAME -p $env:PYPI_PASSWORD --repository-url https://upload.pypi.org/legacy/ --skip-existing dist/*" 2>&1 | Out-Null
       }
+  - "conda deactivate"
+  - "conda env remove -n %ENV_NAME%"
```
Comment (megies) [inline appveyor.yml]: I think this should go into `on_finish:` section so its always executed

Comment (megies) [reply appveyor.yml]: https://www.appveyor.com/docs/appveyor-yml/

File: appveyor.yml (hunk 1 above)
Comment (ThomasLecocq) [reply appveyor.yml]: :-) yep

File: appveyor.yml (hunk 2 above)
Comment (ThomasLecocq) [reply appveyor.yml]: good point, doing now

**Comment by ThomasLecocq** [comment]
successful build of 4 versions, in parallel on my machine:
https://ci.appveyor.com/project/obspy/obspy/builds/31071147

4 jobs = 10min :-))

**Comment by ThomasLecocq** [comment]
closing this, it was a cool idea, but now with GH actions, it's no longer needed

**Pull request closed without merging**